language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/command/testgen/template/AbstractTestTemplate.java

```diff
@@ -0,0 +1,145 @@
+package org.ballerinalang.langserver.command.testgen.template;
```

Comment:
Missing License Header

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/11447